R/1.0_module_data.R

```diff
@@ -39,9 +39,10 @@ NULL
 ui_data <- function(id, data, title, header, footer) {
   ns <- shiny::NS(id)
   shiny::div(
+    id = ns("teal_data_body"),
```

Comment:
can be just `body` or preferably `content` as `ui_data` is already called with namespace `teal-data`, so `teal-data-content` is shorter than `teal-data-teal_data_body`.
```suggestion
    id = ns("content"),
```